### ISSUE TITLE
Fisheye/Panorama not flipped

### ISSUE THREAD

**Issue by RoySalisbury** [aaronwmorris/indi-allsky#1257]
I have both "Flip Image Vertically" and "Flip Image Horizontally" enabled for my camera.  However, it does not look like the panorama generation takes these settings into account. 

![Screenshot 2024-05-11 at 9 54 38 PM](https://github.com/aaronwmorris/indi-allsky/assets/12501943/6f5c4b5c-a2fc-4592-95b0-25b13fca64db)

Also, as a "feature" it might be nice to add the cardinal markers (N/S/E/W) at the bottom of the image.

Thanks.

**Comment by aaronwmorris** [comment]
The panorama is generated after the image is flipped, so the flipped image is used.

I will look into adding the cardinal directions to the panorama.

**Comment by RoySalisbury** [comment]
I guess it just looks odd.  I have the panorama set so that the center is looking south.  IN that case, I would expect the sun/moon to rise from the left -> right (east -> west).  However, the image is flipped ... It goes from right -> left.

The cardinal directions for this image would be N-W-S-E-N .. and would expect it to be N-E-S-W-N ...

![Screenshot 2024-05-12 at 9 53 08 PM](https://github.com/aaronwmorris/indi-allsky/assets/12501943/0f24ff24-7fba-4ef0-bdc3-57d3434ad8ba)

**Comment by aaronwmorris** [comment]
I never really paid attention to how it was generated.  This is just how the [fish2pano](https://github.com/bluthen/fish2pano) python module works.

**Comment by RoySalisbury** [comment]
I took a look at that project.  In it he has 2 sample images.  From them you can tell that the result is actually a mirror image (the text on the image is mirrored).  

Wonder if you can just do a mirror of the result to get it back to normal.  Add a switch to the config to enable/disable it.

**Comment by aaronwmorris** [comment]
I'm guessing it is just reading the image counter-clockwise.  I'll look into adding an option to flip the image horizontally.

**Comment by aaronwmorris** [comment]
Nevermind, after reviewing the images, its reading them clockwise.

**Comment by aaronwmorris** [comment]
Merged #1264 with the ability to add cardinal directions to panoramas.  There are options to flip the panorama as well.

I had to setup some special test images to figure out details about the panorama.  The first important point was the the starting point of the panorama, which is 90 degrees clockwise and the progression is clockwise from that point.  The image needs to be rotated -90 degrees to get the bottom of the image to be the center of the panorama.
